network/interfaces.sls

```diff
@@ -12,7 +12,17 @@
   {%- endif -%}
 {%- endmacro -%}
 
-{% if salt['pillar.get']('network:interfaces', False) %}
+{%- if 'dhclient_conf' in salt['pillar.get']('network') %}
```

Comment:
please change to

``{%- if 'dhclient_conf' in salt['pillar.get']('network', {}) %}``

for the case the ``network`` pillar is undefined.